utilities/operator.py

```diff
@@ -531,7 +532,7 @@ def wait_for_mcp_update_completion(machine_config_pools_list, initial_mcp_condit
 def wait_for_all_nodes_ready(nodes):
     nodes_not_ready = None
     sampler = TimeoutSampler(
-        wait_timeout=TIMEOUT_5MIN,
+        wait_timeout=TIMEOUT_15MIN,
```

Comment:
This is a significant update to wait time. Please add argument and keep the original value as default.